cpp/CMakeLists.txt

```diff
@@ -190,6 +190,10 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
     "Build with zstd compression"
     ON)
 
+  option(ARROW_WITH_GRPC
+    "Build with GRPC"
+    ON)
```

Comment:
Could you leave this off by default for now?

Reply:
@wesm , sure, I'm wondering regarding addressing new dependencies issues. At least manylinux (https://travis-ci.org/MaxRis/arrow/jobs/284387240) image build is failing now, seems, because of missed GoLang.

Reply:
Right, we don't need to resolve this right now. I will verify locally that the build works on a machine with Go installed. We'll need to do some work, I imagine, to get GRPC tests running in Travis CI -- it may be easier to build GRPC in conda-forge and use a toolchain library